src/components/Nav/Navigation.tsx

```diff
@@ -99,41 +101,52 @@ class Navigation extends React.Component<PropsType, StateType> {
   };
 
   render() {
+    let authenticated = sessionStorage.getItem('user');
```

Comment:
I have one security comment here. So, authentication state is stored in sessionStorage. Is it possible to add this step from the login page and bypass this ? If I understood, with the debug tools of redux, I can locally submit an item called 'user' and that would be make the page render ?

Reply:
The idea is in the next steps store this in redux. Because we need username and password for all the API calls I opened https://issues.jboss.org/browse/KIALI-763 for this, the next step is prepare the backend to make the authentication through openshift/kubernetes and work with a token.